pkg/cloudprovider/provider/anexia/instance.go

```diff
@@ -45,6 +45,11 @@ func (ai *anexiaInstance) ID() string {
 	return ai.info.Identifier
 }
 
+// TODO(xmudrii): Implement this.
+func (ai *anexiaInstance) ProviderID() string {
+	return ""
+}
```

Comment:
I wasn't able to get Anexia cluster working, so this is left as a TODO once we can provision Anexia clusters.